src/adapters/rubicon.js

```diff
@@ -51,24 +54,28 @@ function RubiconAdapter() {
       try {
         ajax(buildOptimizedCall(bid), bidCallback, undefined, {withCredentials: true});
       } catch(err) {
-        utils.logError('Error sending rubicon request for placement code ' + bid.placementCode, null, err);
+        if(!(err instanceof RubiconError)) {
+          throw err;
+        }
+        utils.logError('Error sending rubicon request for placement code ' + bid.placementCode);
```

Comment:
this only logs if there are no sizes (the only `RubiconError` thrown).  What's the rational for handling only that exception here and propagating all others up the chain?  Also, this doesn't log the specific `err`